### PR TITLE
Forum

### LINKED ISSUE #43: [Website] Create Forum Post
Being able to create a forum post.

### PR DESCRIPTION
**Pull request by Bretl7**
Fixed #43
Ability to post comments on threads

### CODE REVIEW

File: Website/Pages/Thread/Thread.razor [hunk 1]
```diff
@@ -57,6 +58,85 @@
...
+                            </div>
+                        </div>
+
+
+                        <div class="form-group row">
+                            <label class="col-sm2 col-form-label" for="Avatar">HERO:</label>
+                            <div class="col-sm-5">
+                                <InputText class="form-control" @bind-Value="tUser.Avatar" />
+                            </div>
+                        </div>*@
```
Comment (cptchuckles) [inline Website/Pages/Thread/Thread.razor:133]:
```suggestion
```

**Review by cptchuckles — CHANGES_REQUESTED** [review]
Delete stuff that isn't being used

**Comment by Bretl7** [comment]
Deleted stuff that isn't used.
I deleted further code beyond your suggested changes and now it won't let me commit suggested changes.

**Comment by cptchuckles** [comment]
That's fine, it's because the suggested changes is a diff aimed at the original version of the files, which doesn't exist, but the changes you made manually already satisfy what I was interested in

**Pull request merged**